src/js/actions/policy.js

```diff
@@ -35,40 +35,73 @@ define(function (require, exports) {
         KeyboardEventPolicy = EventPolicy.KeyboardEventPolicy;
 
     /**
-     * Install a new policy list.
+     * The previous policies during temporary policies disabling
      *
-     * @private 
-     * @param {string} kind A value defined in PolicyStore.eventKind
-     * @param {Array.<KeyboardEventPolicy>} policies
-     * @return {Promise}
+     * @type {?EventPolicySet}
      */
-    var _addPolicies = function (kind, policies) {
+    var _cachedKeyboardPolicies = null;
```

Comment:
This state needs to be reset to null in `onReset`. Sometimes I think we should actually bar state in actions entirely... Another day!